### ISSUE TITLE
Screenreader gives vague information for prev, next buttons

### ISSUE THREAD

**Issue by jonels-msft** [readthedocs/sphinx_rtd_theme#954]
### Problem

Although the buttons have informational tooltip text, that text is not announced by the screenreader. Instead it says, "link n, alt plus n" and "link p, alt plus p."

### Reproducible Project

https://docs.citusdata.com/en/v9.3/develop/migration.html

#### Error Logs/Results

Here is a video demonstrating the problem: https://www.youtube.com/watch?v=-tylvGPHwF4

#### Expected Results

In scan mode the screenreader should announce e.g. the previous link as something like, "Previous link choose distribution column press alt + p to access the control."

### Environment Info

- Python Version: 3.7.7
- Sphinx Version: 1.4.9
- RTD Theme Version: 0.2.4

**Comment by Daltz333** [comment]
I'm not sure a bug report is valid for such an old version.

**Comment by stsewd** [comment]
Looks like the issue is still valid, the problem is that icon is getting interpreted as text

https://github.com/readthedocs/sphinx_rtd_theme/blob/006cd2162aac1c3e4c00542a38c9e47e0524ca46/sphinx_rtd_theme/breadcrumbs.html#L74-L81

```
<span class="fa fa-arrow-circle-left"></span>
```
